utils/dbz_event_to_fhir_config.json

```diff
@@ -197,7 +197,8 @@
       "enabled": "true",
       "title": "Patient",
       "parentTable":"person",
-      "joinClause":"SELECT uuid FROM person,patient WHERE person_id = patient_id",
+      "parentForeignKey": "person_id",
+      "childPrimaryKey" : "patient_id",
```

Comment:
@gitcliff , you can do the same  for all the tables without uuid ...and we see if it can work out